### PR TITLE
[EOC-208] Add react intl to project

### PR DESCRIPTION
**Pull request by adamklepacz**
React-intl added.
Footer translated for test purpose.

All translations inside locales/en.json file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
   "repository": "github:sandstreamdev/eoc",
   "dependencies": {
     "@sendgrid/mail": "6.4.0",
+    "babel-plugin-react-intl": "3.2.0",
```
Comment (marekrozmus) [inline package.json]: remove this as it is not used anymore

File: package.json [hunk 2]
```diff
@@ -58,7 +60,9 @@
     "validator": "11.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "7.4.4",
     "@babel/core": "7.4.5",
+    "@babel/node": "7.4.5",
```
Comment (marekrozmus) [inline package.json]: are these added dev dependecies still used?

File: client/src/app/components/Layout/Toolbar/components/UserBar/UserBar.jsx [hunk 3]
```diff
@@ -77,7 +77,7 @@ class UserBar extends Component {
           >
             <ul className="user-bar__menu">
               <li className="user-bar__menu-item">
-                {`Logged as:  ${name}`}
+                {`Logged as: ${name}`}
```
Comment (marekrozmus) [inline client/src/app/components/Layout/Toolbar/components/UserBar/UserBar.jsx]: is this change relevant to this PR?

File: .gitignore [hunk 4]
```diff
@@ -67,3 +67,6 @@ typings/staticFiles
 dist
 .DS_Store
 .stylelintcache
+
+#messages translations generated by react-intl
+client/src/locales/messages/
```
Comment (marekrozmus) [inline .gitignore]: is this still valid after we remove the auto generating script?

File: .babelrc [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
-    "@babel/preset-react"
-  ],
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
   "plugins": [
+    [
+      "react-intl",
+      {
+        "messagesDir": "./client/src/locales/messages/"
+      }
+    ],
```
Comment (marekrozmus) [inline .babelrc]: is this still valid after we remove the auto generating script?

File: client/src/app/components/Layout/Toolbar/components/UserBar/UserBar.jsx (hunk 3 above)
Comment (adamklepacz) [reply client/src/app/components/Layout/Toolbar/components/UserBar/UserBar.jsx]: not really

File: client/src/app/components/Footer.jsx [hunk 6]
```diff
@@ -8,15 +9,16 @@ const Footer = () => (
...
         </Link>
         <span className="footer__text-divider"> | </span>
         <Link className="footer__text footer__link" to="/privacy-policy">
-          Privacy & Terms
+          <FormattedMessage id="footer.privacy" />
         </Link>
       </span>
       <span className="footer__text footer__copyrights">
-        &copy; Copyrights 2019
+        &copy;
```
Comment (marekrozmus) [inline client/src/app/components/Footer.jsx]: you can move the copyright sign into the translation text

File: .babelrc [hunk 7]
```diff
@@ -1,9 +1,4 @@
...
-  "presets": [
-    "@babel/preset-env",
-    "@babel/preset-react"
-  ],
-  "plugins": [
-    "@babel/plugin-proposal-class-properties"
-  ]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": ["@babel/plugin-proposal-class-properties"]
 }
```
Comment (marekrozmus) [inline .babelrc]: please revert to original formatting so that this file is removed from this PR

File: package.json [hunk 8]
```diff
@@ -58,6 +59,7 @@
     "validator": "11.0.0"
   },
   "devDependencies": {
+    "@babel/cli": "7.4.4",
```
Comment (marekrozmus) [inline package.json]: what is this for?

File: client/src/app/components/Footer.jsx (hunk 6 above)
Comment (adamklepacz) [reply client/src/app/components/Footer.jsx]: I can't, cause react intl renders this as a string

"$copy Copyrights 2019"

File: package.json (hunk 8 above)
Comment (adamklepacz) [reply package.json]: unused, deleted

File: .babelrc (hunk 7 above)
Comment (adamklepacz) [reply .babelrc]: it is reverted, but now it is formatted in a different way. Don't know why, because I still have the same prettier config.
